patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch

```diff
@@ -0,0 +1,24 @@
+--- a/third_party/blink/renderer/bindings/scripts/generate_bindings.py
++++ b/third_party/blink/renderer/bindings/scripts/generate_bindings.py
+@@ -11,6 +11,10 @@ import sys
+ import web_idl
+ import bind_gen
+ 
++# shutil.copy
++import shutil
++import os
++
+ 
+ def parse_options():
+     parser = optparse.OptionParser(usage="%prog [options] TASK...")
+@@ -53,6 +56,10 @@ def parse_options():
+ 
+ 
+ def main():
++    # This fix build issue on Windows caused by missing google dependencies.
++    if os.name == 'nt':
++        shutil.copy('../../third_party/llvm-build/Release+Asserts/bin/clang-format.exe', '../../buildtools/win')
```

Comment:
@AKile9V Doesn't this copy the binary every time this program is invoked? Can we check for existence before copying?

Reply:
Sure, I think that is good idea. I'll try to add some checker if clang-format is already there, test it on my build and then will fill in